website/views.py

```diff
@@ -23,4 +25,24 @@ def get_inventory():
             return render_template("get_inventory.html", products=inventory_products,
                                    categories=unique_categories)
     except Exception as e:
-        logging.exception(e)
\ No newline at end of file
+        logging.exception(e)
+
+@views.route('admin/add-inventory', methods=['POST'])
+@login_required
+def add_inventory():
+    name = request.form.get("name")
+    description = request.form.get("description")
+    quantity = int(request.form.get("quantity"))
+    category = request.form.get("category")
+    weight = int(request.form.get("weight"))
+    expiry_date = datetime.strptime(request.form.get("expiry_date"), '%m-%d-%Y')
+    print(f"Name: {name}, Description: {description}, Quantity: {quantity}, Category: {category}, Weight: {weight}, Expiry Date: {expiry_date}")
+    # Validating the null check of input fields
+    if any(value is None or value == "" for value in (name, description, quantity, category, weight, expiry_date)):
+        flash("One or more form values are missing", category='error')
+        return render_template("login.html", user=current_user)   
```

Comment:
Since add_inventory page is not yet committed, are you redirecting to login.html?

Reply:
Yes, that is right. This has to be replaced with the corresponding add_inventory UI page and provide an access to the admin user to re enter the missing values of the inventory.